teleport-server/templates/cloud-init.yaml.tpl

```diff
@@ -39,6 +39,8 @@ write_files:
         type: dynamodb
         region: ${teleport_dynamodb_region}
         table_name: ${teleport_dynamodb_table}
+        audit_events_uri: [file:///var/lib/teleport/audit/events, dynamo://${teleport_dynamodb_table}_events]
```

Comment:
unsure if this will create files `/var/lib/teleport/audit/events/*.log` or `/var/lib/teleport/audit/events.log`

Reply:
I'm 99% sure it'll be the first option 😅